### PR TITLE
Ace Interaction & customization integration

### PR DESCRIPTION
**Pull request by Ollieollieolllie**
Creating of Misc PBO to House Ace interact  & customization additions
Addition of Ace interact points -
- on the tail to set a five-figure tail number
- on the door handles to open the doors
- on the fire control radar to add and remove It
- on the body of the apache to install IAFS or 1200rnd Magazine

Creation of Empty objects
- Internal Auxilary fuel system 
- 1200 Magazine
- Fire control radar

Fix for Total fuel on fuel page fix

### CODE REVIEW

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_misc/$PBOPREFIX$ [hunk 1]
```diff
@@ -0,0 +1 @@
+fza_ah64_misc
```
[the remark this answers is not in the document]
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_misc/$PBOPREFIX$]: done

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
Still have some outstanding questions from the previous review

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp [hunk 2]
```diff
@@ -1989,7 +1990,7 @@ class CfgVehicles
 			dayLight = 0;
 			drawLight = 1;
 			intensity = 75;
-			useFlare = 0;
+			useFlare = 1;
```
[the remark this answers is not in the document]
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp:1993]: sorry, bad habit of fixing things while adding things
just made them more visible at day

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/event/fn_eventInit.sqf [hunk 3]
```diff
@@ -154,7 +154,7 @@ while {
 }
 do {
     private _battBusOn       = _heli getVariable "fza_systems_battBusOn";
-    if ((isLightOn [_heli,[0]]) && _battBusOn) then {
+    if ((isLightOn [_heli,[0]]) && !_battBusOn) then {
```
[the remark this answers is not in the document]
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/event/fn_eventInit.sqf:157]: sorry, bad habit of fixing things while adding things again
fixed the logic for flood lighting power

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp [hunk 4]
```diff
@@ -2059,7 +2060,7 @@ class CfgVehicles
 			outerAngle = 90;
 			coneFadeCoef = 10;
 			useFlare = 1;
-			flareSize = 10;
+			flareSize = 15;
```
[the remark this answers is not in the document]
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp]: removed

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/weapon/fn_weaponSwapM230Mag.sqf [hunk 5]
```diff
@@ -1,21 +1,16 @@
 params ["_heli", "_phase"];
 
-systemChat str _this;
-
-if (_phase == _heli getVariable "fza_ah64_magazineAddLastValue") exitwith {};
+_heli removeMagazineTurret ["fza_m230_300", [0]];
+_heli removeMagazineTurret ["fza_m230_1200", [0]];
```
[the remark this answers is not in the document]
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/weapon/fn_weaponSwapM230Mag.sqf:4]: yes, the addition of the different magazine causes it to have full ammo.
i could set it to 0 after changing

[the remark this answers is not in the document]
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/weapon/fn_weaponSwapM230Mag.sqf:4]: Gonna set it to 0

**Review by BradMick — APPROVED** [review]
Looks good to me. Nothing glaring stands out.

**Pull request merged**